### PR TITLE
Doc 532 create tables

### PR DESCRIPTION
**Pull request by GodsonLeigh**

### CODE REVIEW

**Review by JungaBunga — CHANGES_REQUESTED** [review]
Database specifications has changed :

Please add "datePublished" column to DAX,TOTALS and ORGANISATIONS tables.

This field will be used to check if records have been published

**Review by JungaBunga — APPROVED** [review]

File: changelog/db.changelog-1.4.xml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+            <column name="actionName" type="VARCHAR(100)"/>
+            <column name="rate" type="VARCHAR(50)"/>
+            <column name="landArea" type="NUMERIC(18,6)"/>
+            <column name="uom" type="VARCHAR(10)"/>
+            <column name="annualValue" type="VARCHAR(50)"/>
+            <column name="quarterlyValue" type="NUMERIC(15,2)"/>
+            <column name="overDeclarationPenalty" type="NUMERIC(15,2)"/>
+            <column name="quarterlyPaymentAmount" type="NUMERIC(15,2)"/>
+            <column name="datePublished" type="DATE">
+                <constraints nullable="true"/>
```
Comment (AbidemiAdio) [inline changelog/db.changelog-1.4.xml:90]: `<constraints nullable="true"/>` line is not necessary

File: changelog/db.changelog-1.4.xml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+                <constraints nullable="false"/>
+            </column>
+            <column name="totalActionPayments" type="NUMERIC(15,2)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="updated" type="DATE">
+                <constraints nullable="false"/>
+            </column>
+            <column name="datePublished" type="DATE">
+                <constraints nullable="true"/>
```
Comment (AbidemiAdio) [inline changelog/db.changelog-1.4.xml:68]: `<constraints nullable="true"/>` not necessary

File: changelog/db.changelog-1.4.xml [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+                <constraints nullable="false"/>
+            </column>
+            <column name="totalQuarterlyPayment" type="NUMERIC">
+                <constraints nullable="false"/>
+            </column>
+            <column name="transDate" type="DATE">
+                <constraints nullable="false"/>
+            </column>
+            <column name="datePublished" type="DATE">
+                <constraints nullable="true"/>
```
Comment (AbidemiAdio) [inline changelog/db.changelog-1.4.xml:22]: `<constraints nullable="true"/>` not necessary

File: changelog/db.changelog-1.4.xml (hunk 2 above)
Comment (GodsonLeigh) [reply changelog/db.changelog-1.4.xml]: removed as suggested

File: changelog/db.changelog-1.4.xml (hunk 3 above)
Comment (GodsonLeigh) [reply changelog/db.changelog-1.4.xml]: removed as suggested

**Review by AbidemiAdio — APPROVED** [review]
QED

**Pull request merged**